src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -50,7 +50,7 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
   return (
     <Box>
       <PasswordInput
-        label={label}
+        label={label + (required ? ' *' : '')}
```

Comment:
Since this is an ODY component, I had to add this asterisk, their component doesnt allow us to remove the label (to implement a custom label) and the ODS rule is that all fields are required by default, see: https://odyssey.okta.design/?path=/docs/mui-components-forms-text-field--password#optional

Not sure how else to work around this without ODS' assistance. 

cc: @lesterchoi-okta